scripts/build_project.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Author: Teghveer Singh Ateliey
+# Date: 2024-10-30
+
+# Check args
+if [ "$#" -ne 4 ]; then
+    echo "Usage: $0 <commit-hash> <repo-path> <project> <platform>"
+    exit
+fi
+
+# Assign inputs to variables
+COMMIT_HASH=$1
+REPO_PATH=$2
+PROJECT=$3
+PLATFORM=$4
+
+# Checks if directory exists
+if [ ! -d "$REPO_PATH" ]; then
+    echo "Error: Repository path '$REPO_PATH' does not exist."
+    exit
+fi
+
+# Navigate to the racecar/firmware directory path
+cd "$REPO_PATH"
```

Comment:
Based on its name, REPO_PATH is the path to `racecar/`, but the makefile is located in `racecar/firmware`. I think you should rename `REPO_PATH` to `RACECAR_PATH` and `cd` into `RACECAR_PATH/firmware`

Reply:
You should make a `FW_PATH` variable from `RACECAR_PATH` and check it exists in the directory check block